Services/CO.CDP.Localization/StaticTextResource.resx

```diff
@@ -3238,4 +3238,34 @@
   <data name="Consortium_ConsortiumEnterSharecode_InvalidCode" xml:space="preserve">
     <value>Share code not found.</value>
   </data>
-</root>
+  <data name="OrganisationRegistration_BuyerDevolvedRegulation_Hint" xml:space="preserve">
+    <value>A contracting authority operating in a devolved nation may be subject to devolved regulations. For example, Northern Ireland, Scotland or Wales.</value>
+  </data>
+  <data name="Supplementary_CookieDetails_TableRow_FT_Auth" xml:space="preserve">
+    <value>Provides Find a Tender with your account details once you’ve signed in</value>
+  </data>
+  <data name="Supplementary_CookieDetails_TableRow_FT_IDP_TIMEOUT" xml:space="preserve">
+    <value>Keeps you logged in to the Supplier Registration Service (used for single sign-on)</value>
+  </data>
+  <data name="Supplementary_CookieDetails_TableRow_FT_PAGE_TIMEOUT" xml:space="preserve">
+    <value>Saves the date/time of the user’s last activity on the site</value>
+  </data>
+  <data name="Supplementary_CookieDetails_TableRow_FT_PAGE_LASTPOLL" xml:space="preserve">
+    <value>Saves the date/time of the last check to see if the user should remain signed in to the site</value>
+  </data>
+  <data name="Supplementary_CookieDetails_TableRow_FT_PAGE_ACTIVE" xml:space="preserve">
+    <value>Saves the outcome of the last check to see if the user should remain signed in to the site</value>
+  </data>
+  <data name="Supplementary_CookieDetails_TableRow_FT_COOKIES_ PREFERENCES_SET" xml:space="preserve">
+    <value>Lets us know that you’ve saved your cookie consent settings</value>
+  </data>
```

Comment:
```suggestion
  <data name="Supplementary_CookieDetails_TableRow_FT_COOKIES_PREFERENCES_SET" xml:space="preserve">
    <value>Lets us know that you’ve saved your cookie consent settings</value>
  </data>
```

Reply:
Flagging this because the key for the localization resource should not contain a space.